codeception/codeception/5.0/tests/Support/Helper/Acceptance.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace Tests\Support\Helper;
+
+// here you can define custom actions
+// all public methods declared in helper class will be available in $I
+
+class Acceptance extends \Codeception\Module
```

Comment:
Please delete helper files.
Codeception 5 don't generate them automatically anymore because they are rarely used.
If user wants to use a helper, they can generate it using `./vendor/bin/codecept generate:helper` command.